bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
+} catch(e) {
```

Comment:
Go ahead. I meant that I'm trying to eliminate that now in my way of writing code in general not in runjs.